syntax = "proto3";

option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.common.dto";
option java_outer_classname = "SecurityProto";
message Security {
	int32 dataType = 1; //股票类型
    string code = 2; //股票代码
}
